src/components/Activity.js

```diff
@@ -0,0 +1,8 @@
+export default function Activity({ key, activityName, children }) {
+  return (
+    <li key={key} id={key}>
```

Comment:
the key only matters if you are going to map. In that case use the key-prop on the Komponent Tag, not inside the component.